### DISCUSSION TITLE
Can't see any heatmap. Do I need to enable/install something else also?

### DISCUSSION THREAD

**Discussion by lamyergeier** [stashapp/stash#5470] [Q&A] [answered]
![image](https://github.com/user-attachments/assets/0175a2d0-0f93-42f6-a2e1-3763cbbea30c)

Where are heatmap displayed? Is it superimposed on the video playback as happens on YouTube?

**Comment by DogmaDragon** [comment, marked as answer]
You need to have a .funscript file with the instructions for it to be generated. See https://docs.stashapp.cc/in-app-manual/interactive/
